### PR TITLE
Add ReSharper code style for braces

### PR DESCRIPTION
**Pull request by asbjornu**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/NancyFx/Nancy/pulls) open
- [x] I have verified that I am following the Nancy [code style guidelines](https://github.com/NancyFx/Nancy/blob/45238076ad0b7f6ecabd6bae8469e30458d02efe/CONTRIBUTING.md#style-guidelines)
- [ ] I have provided test coverage for my change (where applicable)

### Description

Configure ReSharper to always force single-line statements to be wrapped in braces, as per the [contributing guidelines](https://github.com/NancyFx/Nancy/blob/master/.github/CONTRIBUTING.md):

> Wrap `if`, `else` and `using` blocks (or blocks in general, really) in curly braces, even if it's a single line.

### CODE REVIEW

File: Nancy.sln.DotSettings [hunk 1]
```diff
@@ -6,7 +6,15 @@
...
+	<s:String x:Key="/Default/CodeStyle/CodeCleanup/Profiles/=NancyStandard/@EntryIndexedValue">&lt;?xml version="1.0" encoding="utf-16"?&gt;&lt;Profile name="NancyStandard"&gt;&lt;CSUseVar&gt;&lt;BehavourStyle&gt;CAN_CHANGE_TO_IMPLICIT&lt;/BehavourStyle&gt;&lt;LocalVariableStyle&gt;ALWAYS_IMPLICIT&lt;/LocalVariableStyle&gt;&lt;ForeachVariableStyle&gt;ALWAYS_IMPLICIT&lt;/ForeachVariableStyle&gt;&lt;/CSUseVar&gt;&lt;CSOptimizeUsings&gt;&lt;OptimizeUsings&gt;True&lt;/OptimizeUsings&gt;&lt;EmbraceInRegion&gt;False&lt;/EmbraceInRegion&gt;&lt;RegionName&gt;&lt;/RegionName&gt;&lt;/CSOptimizeUsings&gt;&lt;CSReformatCode&gt;True&lt;/CSReformatCode&gt;&lt;CSShortenReferences&gt;True&lt;/CSShortenReferences&gt;&lt;CSReorderTypeMembers&gt;True&lt;/CSReorderTypeMembers&gt;&lt;CSMakeFieldReadonly&gt;True&lt;/CSMakeFieldReadonly&gt;&lt;CSCodeStyleAttributes ArrangeTypeAccessModifier="False" ArrangeTypeMemberAccessModifier="False" SortModifiers="False" RemoveRedundantParentheses="False" AddMissingParentheses="False" ArrangeBraces="True" ArrangeAttributes="False" ArrangeArgumentsStyle="False" /&gt;&lt;/Profile&gt;</s:String>
```
Comment (khellang) [inline Nancy.sln.DotSettings]: I think all of

- ArrangeTypeAccessModifier
- ArrangeTypeMemberAccessModifier
- SortModifiers
- RemoveRedundantParentheses
- AddMissingParentheses

Can be true :smile:

Comment (asbjornu) [inline Nancy.sln.DotSettings]: Done.

**Comment by khellang** [comment]
Ah, just remembered one thing. Can you just copy this to the Nancy.Next.sln.DotSetting as well?

**Comment by asbjornu** [comment]
@khellang Done.

**Comment by khellang** [comment]
Weeeee :grin:

**Comment by asbjornu** [comment]
@thecodejunkie Haha, thanks for that. 😄  Adventure Time ❤️

**Pull request merged**